orchestra/todos/urls.py

```diff
@@ -29,3 +26,10 @@
         views.worker_task_recent_todo_qas,
         name='worker_task_recent_todo_qas'),
 ]
+
+router = routers.SimpleRouter()
```

Comment:
awesome! :)

just making sure this doesn't result in any newly necessary QA or exposing any new methods.

Reply:
Addressed. We now have this: `http_method_names = ['get', 'post', 'put']` :)